parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
+
+set -euo pipefail
+
+for kubelet_label in $(echo $KUBELET_NODE_LABELS | sed "s/,/ /g")
+do
+  kubectl label --kubeconfig /var/lib/kubelet/kubeconfig --overwrite nodes $HOSTNAME $kubelet_label
```

Comment:
https://jamesdefabia.github.io/docs/user-guide/kubectl/kubectl_label/

```
# Update pod 'foo' with the label 'status' and the value 'unhealthy', overwriting any existing value.
kubectl label --overwrite pods foo status=unhealthy
```

Reply:
--overwrite[=false]: If true, allow labels to be overwritten, otherwise reject label updates that overwrite existing labels.

Reply:
Not sure if you are saying to drop ```--overwrite```, but I believe we want/need ```--overwrite=true```. The reasoning is that from my understanding we will be updating the value of the labels, i.e. resetSP label, but keeping the name of the label the same when reimages. From my understanding of the ```--overwrite``` value if I set it to false it would reject the update to the value and therefore make this system rather useless. If you think I'm misunderstanding something please correct me